addon/index.js

```diff
@@ -346,9 +346,13 @@ function updateExperiments() {
 function syncAllAddonInstallations(serverInstalled) {
   const availableIDs = Object.keys(store.availableExperiments);
   const clientIDs = Object.keys(store.installedAddons);
-  const serverIDs = serverInstalled
-    .filter(item => item.client_id === store.clientUUID)
-    .map(item => item.addon_id);
+  const serverIDs = [];
```

Comment:
Oof, you know something just occurred to me: After this makes it to prod, we will have a period where folks are running the previous add-on against the site until their Firefox looks for an update. Will the API change break the old version of the add-on in a nasty way? If so, we may need to consider [versioning the API](https://github.com/mozilla/testpilot/issues/53).